Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM maven:3-openjdk-11
+COPY ./src /opt/DockerPracticeJavaApplication/
+COPY ./pom.xml /opt/DockerPracticeJavaApplication/
+WORKDIR /opt/DockerPracticeJavaApplication
+RUN mvn clean compile assembly:single
```

Comment:
Hi @YuriGaluza, have you tried to build this Dockerfile? This RUN part fails with the next error: 
```[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:3.1.1:single (default-cli) on project docker-practice-java: Error reading assemblies: No assembly descriptors found. -> [Help 1]```